.github/workflows/cargo.yml

```diff
@@ -40,6 +40,12 @@ jobs:
     - name: Install Cargo
       if: runner.os == 'Linux'
       run: curl https://sh.rustup.rs -sSf | sh -s -- -y
+    # The cargo config file is required for OneBranch CFS.
+    # Remove it for this workflow. Otherwise, cargo will try to reach the internal feed
```

Comment:
There's got to be a better way...

Reply:
I have seen some proposals here https://internals.rust-lang.org/t/problems-of-cargo-config-files-and-possible-solutions/12987/8 but I have not seen an elegant way to ignore .cargo/config.toml. We could also move the cargo file into a child folder, where cargo would try to locate the config file there.e

Reply:
I'm worried while this fixed **our** repo, other consumers of our crate will be broken by this. Is there no way to tell OneBranch to ignore this?

Reply:
What do you mean by other consumers of our crate will be broken by this?

Reply:
We delete this config file so our crate behaves the same as it originally did. What will be broken?

Reply:
There is a way to tell CFS to ignore everything at repo level by submitting a form.

Reply:
Let's go that route.